sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -32037,87 +32035,6 @@
                       "lazyLoad": false,
                       "autoAdjust": false,
                       "id": "ek4ohwi"
-                    },
-                    {
-                      "label": "<strong>Please estimate your weekly transportation costs.</strong>",
-                      "prefix": "$",
-                      "mask": false,
-                      "spellcheck": true,
-                      "tableView": false,
-                      "currency": "CAD",
-                      "inputFormat": "plain",
-                      "validate": {
-                        "required": true,
-                        "custom": "",
-                        "customPrivate": false,
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false,
-                        "min": "",
-                        "max": "",
-                        "step": "any",
-                        "integer": ""
-                      },
-                      "key": "transportationcostsEstimate",
```

Comment:
This wrong reference is used in data inventory report. 

@andrewsignori-aot @andrepestana-aot @guru-aot @lewischen-aot 

The sql of data inventory reports needs to be handled here. 
![image](https://github.com/bcgov/SIMS/assets/54600590/163d0c34-ab79-4f1f-a0aa-f65f3f1f950c)

![image](https://github.com/bcgov/SIMS/assets/54600590/4fa570cc-097e-4d27-aa1f-e2b2ceb04c85)

Reply:
If it is present in the report and it is obsolete we can assume that it is coming from the wrong place, right?
I would suggest changing the report to output `--TBD--` also.
We also have a spreadsheet mapping this report that can be just updated.
If updating the report is not part of this effort and the report would still be generated without errors I would not consider it as a blocker.

Reply:
Yes, if it's not part of this ticket, that would be good to create one to fix it.

Reply:
As the team talked in the daily meeting, the Team agreement is to not create a ticket right now since this will be reviewed by Ministry.